release-team/role-handbooks/communications/templates/mid-cycle-blog-sneak-peek.md

```diff
@@ -0,0 +1,91 @@
+# Outline for the release mid-cycle blog
+This outline can be used as reference for writing up the mid cycle release blog in case of sneak peek required. Deprecations and removal is required in case there are any.
```

Comment:
```suggestion
This outline can be used as a reference for writing up the mid cycle release blog in case of sneak peek is required. Deprecations and removal are required in case there are any.
```